index.js

```diff
@@ -1,95 +1,128 @@
 // ***** Question 1 *****
 
+counter = 1
```

Comment:
Make sure to use `let` or `const` when you're declaring variables! It's a tough habit to break from Ruby but it's important to remember. For this one, we'd want to use `let`:

```js
// declare counter variable and assign it to 1
let counter = 1
// reassign it to 2
counter = 2
```